src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+  <head>
+    <title>Title</title>
+  </head>
+  <body>
+  <h1>Hello World</h1>
+  </body>
+</html>
```

Comment:
In checklist it's mentioned about one empty line in the end of files

Reply:
I am absolutely sure that I added an empty line in the end of every file. It just does not show an additional line here.

Reply:
<img width="593" alt="Screenshot 2021-01-19 at 20 47 38" src="https://user-images.githubusercontent.com/67925366/105079081-94ec2e80-5a97-11eb-806c-064f9f08b696.png">

Reply:
sorry, then)